utilities/plotting/useLivePlot.m

```diff
@@ -0,0 +1,45 @@
+classdef useLivePlot < handle
+    % Sets up a live plot on a given figure 
+    properties
+        figureId
+        handle
+    end
+
+    properties (Constant)
+        callback = @plotRefSLDHelper
```

Comment:
Will we want to use other plotters in time?

Reply:
No I don't think so